github/github.go

```diff
@@ -241,13 +241,14 @@ func NewClient(httpClient *http.Client) *Client {
 // specified, the value pointed to by body is JSON encoded and included as the
 // request body.
 func (c *Client) NewRequest(method, urlStr string, body interface{}) (*http.Request, error) {
-	rel, err := url.Parse(urlStr)
+	if c.BaseURL.Path != "" && !strings.HasSuffix(c.BaseURL.Path, "/") {
+		return nil, fmt.Errorf("BaseURL must have a trailing slash, but %q does not", c.BaseURL)
```

Comment:
Because I got an error in the test, I checked the sky of Path.
Since the test allowed `http://127.0.0.1:50300`, I thought that an empty pass was allowed.
I think your proposal is correct.
I will fix the test.